eng/tools/tsp-client-tests/tsconfig.json

```diff
@@ -1,6 +1,12 @@
 {
   "extends": "../tsconfig.json",
   "compilerOptions": {
-    "outDir": "./dist",
-  }
+    "composite": true,
```

Comment:
Must be set in sub-projects, but should not be set in default, to avoid potential double-compilation.